### ISSUE TITLE
[Enhancement]: @woocommerce/data Improve plugins installed/activated snackbar notices copy

### ISSUE THREAD

**Issue by vladolaru** [woothemes/woocommerce#53596]
### Describe the solution you'd like

When using `installAndActivatePlugins` with only one plugin, the notice copy should use singular, not plural.

Instead of `Plugins were successfully installed and activated.` we should see `A plugin was successfully installed and activated.`
Even better would be use the plugin title (or slug if it is too complicated) in the copy - something like `WooPayments was successfully installed and activated.`

**Comment by wavvves** [comment]
@barryhughes I believe this was labelled Rubik by mistake 🤔

**Comment by vladolaru** [comment]
Yes. I've assigned it to Moltres since we are the ones asking for it.

**Comment by barryhughes** [comment]
Sorry, thanks for correcting.
